extensions/applicationinsights-properties-js/src/TelemetryContext.ts

```diff
@@ -35,6 +35,17 @@ function _nullResult(): string {
     return null;
 }
 
+function setTageValue(tags: Tags | Tags[], field: string, value: string): void {
+    // Function body
+    if (Array.isArray(tags)) {
+        tags.forEach(tag => setValue(tag, field, value, isString)
```

Comment:
This doesn't look correct... Let me try and repo locally to see the errors.

Reply:
Ok, looking at the usage a bit deeper and the returned error message, I think we do the following
- Remove the `Tags[]` completely so it's just defined as `tags?: Tags;` as during the serialization process in `Serializer.ts` we process this field with the `_serializeStringMap` and if that encounters an "error" it will actually report it as an error (which is what would happen if they do actually pass an incomplete array).
- Add some release notes about this possible breaking change, at the very least lets call this out in the "name" of this PR, so if someone looks at the history it will be obvious

As this change will only affect `npm` users and the correct "typing" is really Tags anyway along with the next release being `4.1.0` its a good time to break anyone how might be trying to pass an array (which they shouldn't).

So